runtime/v2/bundle.go

```diff
@@ -115,16 +117,32 @@ type Bundle struct {
 }
 
 // Delete a bundle atomically
-func (b *Bundle) Delete() error {
+func (b *Bundle) Delete(ctx context.Context) error {
 	work, werr := os.Readlink(filepath.Join(b.Path, "work"))
 	rootfs := filepath.Join(b.Path, "rootfs")
+
+	// on windows hcsshim writes panic logs in the bundle directory in a file named
+	// "panic.log" log those messages (if any).
+	// Read only upto 1MB worth of data from this file. If the file is larger
+	// than that, log that.
+	readLimit := int64(1024 * 1024) // 1MB
+	logBytes, err := cioutil.LimitedRead(filepath.Join(b.Path, "panic.log"), readLimit)
+	if err == nil {
+		if int64(len(logBytes)) == readLimit {
+			log.G(ctx).Warnf("shim panic log file %s is large than 1MB, logging only first 1MB", filepath.Join(b.Path, "panic.log"))
+		}
```

Comment:
Wouldn't this get hit even if the file was exactly 1MB? 

Also nit: *larger

Reply:
Yeah that's right but it is just a warning. I don't think it is a problem if we log this warning for a file that is of exactly 1MB in size rather than adding exact stat call here to check the size of the file. What do you think?

Reply:
I think it's probably fine. Another option is to also return the file size from the `LimitedRead` func.

Reply:
I can add that if you think that that's important.